src/main/java/uk/gov/justice/hmpps/prison/repository/InmateAlertRepository.java

```diff
@@ -234,4 +241,12 @@ public long createNewAlert(final long bookingId, final CreateAlert alert) {
 
         return alertSeq;
     }
+
+
+    private static void checkQueryIsNotUsed(String query) {
+        Optional.ofNullable(query)
+            .filter(not(String::isBlank))
+            .ifPresent(q -> log.warn("Alert repository has received a query parameter that might be vulnerable"));
```

Comment:
Wonder is its worth thinking of putting an aspect around all endpoints with `String query` @andymarke ?